pages/components/card/card.css

```diff
@@ -0,0 +1,83 @@
+/* Importing external styles */
+@import "../../../styles/typography.css";
+@import "../../../styles/variable.css";
+@import "../../../styles/reset.css";
+@import "../../../styles/header.css";
+@import "../../../styles/drawer.css";
+@import "../../../styles/padding.css";
+@import "../../../styles/margin.css";
+@import "../../../styles/utils.css";
+@import "../image/image.css";
+@import "../button/button.css";
```

Comment:
nice work creating these files separate! even i haven't due to time

Reply:
Thank you!